packages/teams-js/src/internal/handlers.ts

```diff
@@ -33,6 +33,12 @@ export function initializeHandlers(): void {
   HandlersPrivate.handlers['beforeUnload'] = handleBeforeUnload;
   pages.backStack._initialize();
 }
+export function uninitializeHandlers(): void {
+  HandlersPrivate.handlers = {};
+  HandlersPrivate.themeChangeHandler = null;
+  HandlersPrivate.loadHandler = null;
+  HandlersPrivate.beforeUnloadHandler = null;
```

Comment:
Should the implementation of clearing the handler state live as a class method on the `HandlersPrivate` class rather than inside this function? You know, encapsulation/abstraction all that good stuff?

(this function can still be exported and called but it should rely on the class to do the actual work?)

Reply:
There are other functions like `initializeHandlers`, `callHandler`, `registerHandler` that are in the same file but outside of the `HandlersPrivate` class. I am wondering if we want to put those inside `HandlersPrivate` class? 
I am moving initializeHandlers and uninitializeHandlers into `HandlersPrivate` because they are very related to each other.